resources/public/api/conf/1.0/endpoints/charitableGiving_retrieve.raml

```diff
@@ -0,0 +1,27 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearRangeInvalid
```

Comment:
The description for the ruleTaxYearRangeInvalid is missing a `The` at the start

Reply:
The other endpoints (delete, put) match to these errors as well and they don't have the "The" at the start. I think it might be best to leave the "The" out for consistency